joomla/joomla-cms
# [plg_system_debug] Allow to hide session tab too

### Summary of Changes

Simple PR to allow also to hide the session tab in debug system plugin.

![image](https://cloud.githubusercontent.com/assets/9630530/18684172/1c38d8d2-7f6b-11e6-955d-f9367f2f6247.png)
### Testing Instructions
1. Apply patch
2. Enable debug in global config and enable debug system plugin
3. In debug sysem plugin set "Show Session Data: No"
4. Load any page and check the session data is not in the debug plugin tabs in the bottom of the page.
### Documentation Changes Required

None.
